src/components/LoginPage/LoginPageComponent.js

```diff
@@ -1,67 +1,175 @@
-                <p>Your mind map is stored as folders &amp; file</p>
-              </div>
-              <img src={slide1} style={{ width: '100%', maxWidth: '750px' }}/>
-            </div>
-            <div className="slide">
-              <h2>Explore subfolders by moving screen around</h2>
-              <div className="slide-text">
-                <p>You can see the entire structure, not just one level</p>
-              </div>
-              <img src={slide2} style={{ width: '100%', maxWidth: '750px' }}/>
-            </div>
-            <div className="slide">
-              <h2>Frequently used folders are BIGGER</h2>
-              <div className="slide-text">
-                <p>Every time you use a folder it becomes bigger</p>
-                <p>Old files you never use get smaller, drowning into deep memory</p>
-              </div>
-              <img src={slide3} style={{ width: '100%', maxWidth: '750px' }}/>
-            </div>
-            <div className="slide">
-              <h2>Single place for notes and files</h2>
-              <div className="slide-text">
-                <p>In NeuralNotes, every folder has a text file for the notes</p>
-                <p>It is displayed in a built-in text editor when a folder is selected</p>
-              </div>
-              <img src={slide4} style={{ width: '100%', maxWidth: '750px' }}/>
-            </div>
-          </div>
-
-          <h2 style={{ textAlign: 'center' }}>Sign up to NeuralNotes for free</h2>
-          <a onClick={authHandler} className="btn btn-social btn-lg btn-google"
-             style={{ width: '250px', display: 'block', margin: '1em auto' }}>
-            <span className="fa fa-google"/>
-            Sign up with Google
-          </a>
-        </div>
-      </div>
+      <Main>
+        <Section>
+          <Slide1 />
+          <Slide2 />
+          <Slide3 />
+          <Slide4 />
+          <Slide5>
+            <H3>Frequently used notes
+              <br />
+              get bigger
+            </H3>
+            <Container>
+            <Tooltip className='tooltip-right'>
+              OLd unused notes
+                <br/>
+                are 
+                <span> small</span>
+                <br />
+                <ArrowDown src={arrow} />
+            </Tooltip>
+            <Figure>
+              <Img src={slide5} />
+            </Figure>
+            <Tooltip className='tooltip-left'>
+                <ArrowUp src={arrow} />
+                <br />
+              Recent, frequently
+                <br />
+                used notes are 
+                <span> big</span>
+            </Tooltip>
+            </Container>
+          </Slide5>
+          <Footer>
+            <H3><span>Neural</span>Notes is open source and free</H3>
+            <Button onClick={authHandler}><span><img src={iconGoogle} /></span>Sing up with Google</Button>
+          </Footer>
+        </Section>
```

Comment:
Having the whole page in one section sounds weird.
Section = any of the more or less distinct parts into which something is or may be divided or from which it is made up.
If we have one section => we divide the page into 1 section? Doesn't make sense IMHO.
If we had at least 2 sections it would be ok. But dividing something into one part... what does it mean?

Reply:
Yes, we could use tag <main>. What do you think about this?

Reply:
>Yes, we could use tag

Which tag do you mean?

Reply:
Sorry, I mean `main`

Reply:
We already have `Main` tag, it's the parent of `Section`.

Reply:
Even don't know what to say... We just need two block elements one in another... The second one don't have any especial sense, how I think... Maybe we can call it `Container`, because it contain other elements and give them an appropriative width? Or just `Div`?

Reply:
Following [an html5 semantic guide](https://www.semrush.com/blog/semantic-html5-guide/), it looks like it should be
```
<main>
  <article>
     content
  </article>
</main>
```

And each slide should be a `<section>`, like we have it right now.

Reply:
Thank you! Very useful article, especially thanks for visualization. Some moments really confused me when I read the documentation and than try to apply it in real project... 😅

Reply:
Fixed in commit [6bf3ca4](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/6bf3ca4e2480350a24ae82118401ca00785791c0)